admin/app/styles/authenticated/users/list.scss

```
@@ -1,11 +1,21 @@
+.pix-select {
```

Ember ne scopant pas le css, je recommande de mettre ces changements dans une classe css pour éviter de surcharger les PixSelect dans les autres pages de l'application.

```suggestion
.user-list {
  .pix-select {